source/textInfos/__init__.py

```diff
@@ -13,11 +13,16 @@
 from abc import abstractmethod
 import weakref
 import re
+from typing import Any, Union, List
+
 import baseObject
 import config
 import controlTypes
 import locationHelper
 
+
+SpeechSequence = List[Union[Any, str]]

```

Comment:
Probably al
```suggestion
from speech.types import SpeechSequence
```

Reply:
I think doing this results in a circular import